.circleci/config.yml

```diff
@@ -26,6 +26,9 @@ jobs:
               sudo npm install -g jsdoc-to-markdown
               sudo pip install git+https://github.com/rahulrrixe/parinx.git@master
               chmod +x ./Tests/scripts/*
+      - add_ssh_keys:
+          fingerprints:
+              - "02:df:a5:6a:53:9a:f5:5d:bd:a6:fc:b2:db:9b:c9:47"
```

Comment:
Why not taking from conf? we should not have this here, remove this since we are in public repo, I think you should also delete the commit history....

Reply:
there is no other way.. this step doesn't work with variables and its just a fingerprint for pem